interceptor/main.go

```
@@ -91,3 +93,16 @@ func main() {
 func readinessHandler(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
+
+func loadPDSignatures() ([]string, error) {
+	var signatures []string = make([]string, 1)
```

golangci-lint (2.12.2)

[error] QF1011: could omit type []string from declaration; it will be inferred from the right-hand side

(staticcheck)